src/modules/Packages.rb

```diff
@@ -18,6 +18,10 @@ class PackagesClass < Module
     RESOLVABLE_TYPES = [:product, :patch, :package, :pattern, :language]
     # Minimum set of packages required to enable VNC server
     VNC_BASE_PACKAGES = ["xorg-x11", "xorg-x11-Xvnc", "xorg-x11-fonts", "xinetd"]
+    # Additional packages needed to run second stage in graphical mode
+    # Use exact package name (i.e. libyui-qt6 instead of libyui-qt) to prevent
+    # the UI from asking for confirmation
+    AUTOYAST_X11_PACKAGES = ["libyui-qt6", "yast2-x11"]
```

Comment:
I'd avoid changing the `SelectSystemPackages`, it calls [`SelectSystemPackages`](https://github.com/yast/yast-packager/blob/master/src/modules/Packages.rb#L1137) which is quite complex and changing the simple `DoProvide` based on package names to something else could possibly break something.

I'd rather change `Packages.vnc_packages` to evaluate the provides and return the evaluated package name. This change is better located with smaller impact on the existing code.